src/routes/safe/components/Balances/SendModal/screens/SendCollectible/index.tsx

```diff
@@ -51,6 +51,7 @@ const SendCollectible = ({ initialValues, onClose, onNext, recipientAddress, sel
   const [selectedEntry, setSelectedEntry] = useState({
     address: recipientAddress || initialValues.recipientAddress,
     name: '',
+    ensToAddress: '',
```

Comment:
Instead of adding a new prop, doesn't make sense to show the ENS domain as the name and also show de converted address so it's easier to verify?